contracts/solidity/oz/TransparentUpgradeableProxy/MyCustomTransparentUpgradeableProxy.sol

```diff
@@ -0,0 +1,10 @@
+//SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.20;
+
+import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+
+//Wrap the OZ contract so I can instantiate it
```

Comment:
Could this comment be tailored to better suit the end user? I believe our repository serves as an example for others to reference, so comments like this may be appeared as a little less professional in my opinion